client/lib/wpcom-undocumented/lib/me.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Module dependencies.
  */
-import Me from 'wpcom/dist/lib/me';
+import Me from 'wpcom/build/lib/me';
```

Comment:
It's lame to rely on an implementation detail like this. Instead, we should export the `Me` constructor from the `wpcom` core require, so that we can do `import { Me } from 'wpcom'`.